public/app/plugins/datasource/graphite/components/GraphiteTextEditor.tsx

```diff
@@ -19,6 +19,13 @@ export function GraphiteTextEditor({ rawQuery }: Props) {
     [dispatch]
   );
 
+  const pasteQuery = useCallback(async () => {
+    const query = await navigator.clipboard.readText();
+
+    dispatch(actions.updateQuery({ query }));
+    dispatch(actions.runQuery());
```

Comment:
Should this always be here? In Eplore we don't run queries automatically but user should click run query and I think that should also be the case when you copy paste. In dashboard we still want to run query onBlur though.